### DISCUSSION TITLE
How to enable pop out image

### DISCUSSION THREAD

**Discussion by ckamps** [gethinode/hinode#777] [Help]
I have some images that I'd like to display and enable users to click on them to expand the size of the image so that they can see more detail. When using the `image` component, a user can right-click and open the image in a new tab.

I'm wondering if there's a means to enable users to simply click on an image to have it expand.

**Comment by markdumay** [comment]
You might want to consider a custom image gallery. Myrthos has created an [example](https://myrthos.net/blog/mgallery/) based on Lightbox 2 that could be a starting point.

**Comment by Aman-Aalam** [comment]
Yeah I ended up some quick writing custom code for this, which I had used in other Hugo sites too.

My use case was simple - enlarge an image that user clicked, no gallery.

**Comment by giomf** [comment]
Would love to see this feature for the Carousel component.
